varsim.py

```diff
@@ -21,6 +24,51 @@
 REQUIRE_VARSIMJAR = not os.path.isfile(DEFAULT_VARSIMJAR)
 if REQUIRE_VARSIMJAR: DEFAULT_VARSIMJAR = None
 
+def convertCN(filenames, operation):
+    """
+    convert '2/1'-like copy number to a single number(e.g. 2)
+    0 will be considered same as 1
+    by default the max number will be kept
+    the change is in place
+    """
+    if operation != "two2one" and operation != "one2two":
+        raise ValueError("Only two2one or one2two allowed")
+    two2one = operation == "two2one"
+    for name in filenames:
+        file = open(name, "r")
```

Comment:
Please do not use `file` as it is a known type in Python. You can try something like `with open(name, "r") as file_fd:` . Using `with` block will also nicely close the file when the block is exited.